mule-gateway/modules/ROOT/pages/mule-gateway-capabilities-mule4.adoc

```diff
@@ -122,9 +122,9 @@ This defines a resource-level pointcut for your API. In this example, the policy
 
 == Enabling Mule Gateway capabilities
 
-To take advantage of Mule Gateway capabilities, you need to establish a communication channel between the Mule runtime and an organization created in a particular plane (ie. US control plane, EU control plane) or Mule Private Cloud Edition.
+To take advantage of Mule Gateway capabilities, you need to establish a communication channel between Mule runtime and an organization created in a particular plane (ie. US control plane, EU control plane) or Mule Private Cloud Edition.
```

Comment:
```suggestion
To take advantage of Mule Gateway capabilities, you need to establish a communication channel between Mule runtime and an organization created in a control plane (US or EU) or Mule Private Cloud Edition.
```

Reply:
we don't believe we allow "i.e."

Reply:
I don't think we look favorably on "you need to" anymore. See here and else where.